source/components/molecules/CaseCard/CaseCard.stories.tsx

```diff
@@ -158,3 +137,38 @@ const SummaryExamples = () => (
     />
   </FlexContainer>
 );
+
+const BookingExamples = () => (
+  <FlexContainer>
+    <Title>Bokning skapad</Title>
+    <CaseCard
+      colorSchema="red"
+      title={`Planerad insats: \nBoendestöd med Ditte`}
```

Comment:
I understand what you are trying to do, but this should be done with code instead.

Reply:
Fixed. Now using the already existing `largeSubtitle` prop for the subtitle.